src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -57,6 +57,8 @@ public class CommandTestUtil {
     public static final String VALID_APPOINTMENT_TYPE_BOB = "Health Check-up";
     public static final String VALID_APPOINTMENT_NOTE_AMY = "Fractured her ankle";
     public static final String VALID_APPOINTMENT_NOTE_BOB = "Yearly check in";
+    public static final String VALID_APPOINTMENT_MARK_AMY = "true";
```

Comment:
Not sure if this is still required after you changed it to boolean?

Reply:
ahh okay thanks for noticing!! i didn't realise, forgot to change that